backend.native/tests/runtime/basic/initializers7.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package runtime.basic.initializers7
+
+import kotlin.test.*
+
+import kotlin.random.Random
+
+object A {
+    val a1 = Random.nextInt(100)
```

Comment:
Using `Random` makes the test non-deterministic.

Reply:
You mean if `a1` and `a2` become the same number and the test would fail to catch a potential bug? Fair enough, is there another way to dynamically initialize `a1` while ensuring compiler won't ever optimize it away?

Reply:
I see.
Well, IIRC, the compiler doesn't optimize the dynamic initialization even if the initializer is literal.
This can likely be verified by more assertions (checking that values are zero in initial state).